exasol_script_languages_developer_sandbox/templates/vm_bucket_cloudformation.jinja.yaml

```diff
@@ -46,13 +44,81 @@ Resources:
                   - ec2:Describe*
                 Resource:
                   - "*"
+  S3CDNBucketPolicy:
+    Type: 'AWS::S3::BucketPolicy'
+    Properties:
+      Bucket: !Ref VMBucket
+      PolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Action:
+              - 's3:GetObject'
+            Principal:
+              Service: "cloudfront.amazonaws.com"
+            Resource:
+              - !Sub "arn:aws:s3:::${ VMBucket }/slc_developer_sandbox/*"
+            Condition:
+              StringEquals:
+                'AWS:SourceArn': !Sub "arn:aws:cloudfront::${AWS::AccountId}:distribution/${CfDistribution}"
+
+  CfDistribution:
+    Type: 'AWS::CloudFront::Distribution'
+    Properties:
+      DistributionConfig:
+        Comment: 'A simple distribution with an S3 origin'
+        DefaultCacheBehavior:
+          AllowedMethods:
+            - 'HEAD'
+            - 'GET'
+          CachedMethods:
+            - 'HEAD'
+            - 'GET'
+          Compress: false
+          DefaultTTL: 86400
+          ForwardedValues:
+            Cookies:
+              Forward: 'none'
+            Headers:
+              - 'Origin'
+            QueryString: false
+          MaxTTL: 31536000
+          MinTTL: 86400
+          TargetOriginId: !Sub 's3-origin-${ VMBucket }'
+          ViewerProtocolPolicy: 'redirect-to-https'
+        DefaultRootObject: 'index.html'
+        Enabled: true
+        HttpVersion: 'http1.1'
+        IPV6Enabled: true
+        Origins:
+          - DomainName: !GetAtt VMBucket.DomainName
+            Id: !Sub 's3-origin-${ VMBucket }'
+            OriginPath: ''
+            OriginAccessControlId: !Ref OAC
+            S3OriginConfig:
+              OriginAccessIdentity: ''
```

Comment:
This is a bit weird. This field apparently is mandatory. If removed, AWS complains when updating the CF stack.
The documentation states:
```
If you want viewers to be able to access objects using either the CloudFront URL or the Amazon S3 URL, specify an empty OriginAccessIdentity element.
```
However, I tested, and the S3 Bucket objects are not accessibly via the S3 URL (which makes sense as  access control of the bucket is set to "private")

Reply:
maybe put this comment to this line

Reply:
Done